src/Monitor/Monitor/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 
 ## Upcoming Release
+* Added new cmdlets:
```

Comment:
Could you add the intention of those change? For instance, "Added new cmdlets to support DiagnosticSetting in more flexible way"